src/convertToObject.js

```diff
@@ -3,16 +3,27 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
   // write your code here
+  const stringToObject = sourceString.split(';').map(style =>
+    style.trim().split(':')).filter(word => word[0] !== '')
+    .reduce((obj, [key, value]) => {
```

Comment:
<img width="1049" alt="image" src="https://github.com/mate-academy/js_style-to-object/assets/86789161/e303e088-957b-4a8d-bd79-8bf29f6603b5">